codec-http/src/main/java/io/netty/handler/codec/http/HttpUtil.java

```diff
@@ -60,12 +60,13 @@ public static boolean isAsteriskForm(URI uri) {
     /**
      * Returns {@code true} if and only if the connection can remain open and
      * thus 'kept alive'.  This methods respects the value of the.
+     *
      * {@code "Connection"} header first and then the return value of
      * {@link HttpVersion#isKeepAliveDefault()}.
      */
     public static boolean isKeepAlive(HttpMessage message) {
         CharSequence connection = message.headers().get(HttpHeaderNames.CONNECTION);
-        if (connection != null && HttpHeaderValues.CLOSE.contentEqualsIgnoreCase(connection)) {
+        if (HttpHeaderValues.CLOSE.contentEqualsIgnoreCase(connection)) {
```

Comment:
@amizurov can you add comment that passing null into `contentEqualsIngoreCase` is fine ?

Reply:
If we passed null to ```contentEqulasIgnoreCase(...)``` this always return false, null safe method.